src/ol/interaction/Extent.js

```diff
@@ -38,7 +38,7 @@ import {createEditingStyle} from '../style/Style.js';
 const ExtentEventType = {
   /**
    * Triggered after the extent is changed
-   * @event ExtentEventType#extentchanged
+   * @event module:ol/interaction/Extent~ExtentEvent#extentchanged
```

Comment:
Even if we do make another change to address the `longname` generation, we need to make a change here too.  We want the docs pointing to `ExtentEvent` instead of `ExtentEventType`.

Reply:
Addressed in 9c6dbe5a339744e990bbb85c775515de6cf8bebb.